src/containers/GroupMenuContainer.js

```diff
@@ -4,7 +4,8 @@ import BookmarkIcon from '@material-ui/icons/Bookmark';
 import BlockIcon from '@material-ui/icons/Block';
 import ModeCommentIcon from '@material-ui/icons/ModeComment';
 import EditIcon from '@material-ui/icons/Edit';
-import {GroupedMenu, generateMenuData, generateMenuItem, InvalidatedIcon, CheckIcon} from 'tc-ui-toolkit';
+import CheckIcon from "@material-ui/icons/Check";
```

Comment:
This was an intentional change so that we use the fatter check icon we had before.
I created a new react svg icon in the toolkit so we could reuse it like a proper react component.

Reply:
Ok. I don't see CheckIcon being exported anywhere from `tc-ui-toolkit`. Am I missing something?
It was not defined on the import that's why I changed it.